test/data/manifests/rhel_7-x86_64-qcow2_customize-boot.json

```diff
@@ -146,6 +146,22 @@
             "user": 0,
             "group": 0
           }
+        ],
```

Comment:
Ah I think I missed a step for the rhel implementation. The yum repo stage is not being added to the generated manifest.

Reply:
The `imageConfig.YUMRepos` were overwriting the `osc.YUMRepos`